doc/testing.md

```diff
@@ -64,6 +64,51 @@ jtreg:$(TOPDIR)/test/nashorn:tier1 jtreg:$(TOPDIR)/test/jaxp:tier1`. You can
 always submit a list of fully qualified test descriptors in the `TEST` variable
 if you want to shortcut the parser.
 
+### Common Test Groups
+
+In the ideal world, contributors would be advised to run all the tests for every change. But in
+real world, one could only be expected to run as many tests as practical, while being mindful
+of the scope for the change, the testing resources available, etc.
+
+The source tree currently defines a few common test groups in the relevant `TEST.groups`
+files. There are test groups that cover a specific component, for example `hotspot_gc`.
+It is a good idea to look into `TEST.groups` files to get a sense what tests are relevant
+to a particular JDK component.
+
+Component-specific tests may miss some unintended consequences of a change, so other
+tests should also be run. Again, it might be impractical to run all tests, and therefore
+_tiered_ test groups exist. Tiered test groups are not component-specific, but rather cover
+the significant parts of the entire JDK.
+
+Multiple tiers allow balancing test coverage and test costs. Lower test tiers are supposed to
+contain the simpler, quicker and more stable tests. Higher tiers are supposed to contain
+progressively more thorough, slower, and sometimes less stable tests, or the tests that require
+special configuration. When choosing which tiers to run, contributors are advised to run the
+first N tiers they can afford to run.
+
+A brief description of the tiered test groups:
+
+- `tier1`: This is the lowest test tier. Multiple developers run these tests every day.
+Normally, at least this tier should be clean before integration. Because of the widespread use,
+the tests in `tier1` are carefully selected and optimized to run fast, and to run in the
+most stable manner. The test failures in `tier1` are usually followed up on quickly, either
+with fixes, or adding relevant tests to problem list. [GitHub Actions workflows](../.github/workflows/),
```

Comment:
I don't think the link to the `.github` directory makes sense here. If there should be a link at all, it should point to information on how GHA works and how to enable these. And frankly I'm not sure if any such comprehensive documentation exists. :-( Perhaps in the Skara wiki? Otherwise the link is just distracting.

Reply:
Sure, I actually put the link there very optimistically :) Dropped it in the new commit.